core/src/avm1/object/color_transform_object.rs

```diff
@@ -43,50 +52,42 @@ impl<'gc> ColorTransformObject<'gc> {
             gc_context,
             ColorTransformData {
                 base: ScriptObject::new(gc_context, proto),
-                params: ColorTransformParams {
-                    red_multiplier: 0.0,
-                    green_multiplier: 0.0,
-                    blue_multiplier: 0.0,
-                    alpha_multiplier: 0.0,
-                    red_offset: 0.0,
-                    green_offset: 0.0,
-                    blue_offset: 0.0,
-                    alpha_offset: 0.0,
-                },
+                red_multiplier: 0.0,
+                green_multiplier: 0.0,
+                blue_multiplier: 0.0,
+                alpha_multiplier: 0.0,
+                red_offset: 0.0,
+                green_offset: 0.0,
+                blue_offset: 0.0,
+                alpha_offset: 0.0,
             },
         ))
     }
 
     add_field_accessors!(
-        [set_params, get_params, params, ColorTransformParams],
-        [
-            set_red_multiplier,
-            get_red_multiplier,
-            params.red_multiplier,
-            f64
-        ],
+        [set_red_multiplier, get_red_multiplier, red_multiplier, f64],
         [
             set_green_multiplier,
             get_green_multiplier,
-            params.green_multiplier,
+            green_multiplier,
             f64
         ],
         [
             set_blue_multiplier,
             get_blue_multiplier,
-            params.blue_multiplier,
+            blue_multiplier,
             f64
         ],
         [
             set_alpha_multiplier,
             get_alpha_multiplier,
-            params.alpha_multiplier,
+            alpha_multiplier,
             f64
         ],
-        [set_red_offset, get_red_offset, params.red_offset, f64],
-        [set_green_offset, get_green_offset, params.green_offset, f64],
-        [set_blue_offset, get_blue_offset, params.blue_offset, f64],
-        [set_alpha_offset, get_alpha_offset, params.alpha_offset, f64],
+        [set_red_offset, get_red_offset, red_offset, f64],
+        [set_green_offset, get_green_offset, green_offset, f64],
+        [set_blue_offset, get_blue_offset, blue_offset, f64],
+        [set_alpha_offset, get_alpha_offset, alpha_offset, f64],
```

Comment:
While we're here, can we rename the getters to `red_multiplier` etc. to match Rust naming conventions?

Reply:
Done.